bin/.gitignore

```diff
@@ -1 +1,6 @@
+<<<<<<< Updated upstream
 .classpath
+=======
+/events/
+/states/
+>>>>>>> Stashed changes
```

Comment:
I'm not sure what this stuff is denoting? If it's from a merge should it be deleted?

Reply:
I hope we won't mess anything up by deleting this. (Stu**d gitignore...) I can delete it though.

Reply:
Def don't delete the whole file. Only the merge stuff like:
```suggestion
.classpath

```